ntoskrnl/io/iomgr/deviface.c

```diff
@@ -1060,7 +1456,7 @@ IoRegisterDeviceInterface(IN PDEVICE_OBJECT PhysicalDeviceObject,
         &ObjectAttributes,
         0, /* TileIndex */
         NULL, /* Class */
-        REG_OPTION_VOLATILE,
+        REG_OPTION_NON_VOLATILE,
```

Comment:
This additional change (and others same) fixes MS sysaudio loading failure with status 0xc0000181 (although the sound is still not working). At booting, it tries to register a new device interface and create some non-volatile registry keys under these parent keys (so they obviously should be non-volatile). TODO: move it into a separate PR.

Reply:
Done. Will revert this here. :slightly_smiling_face:

Reply:
Reverted. :ok_hand: